CONTRIBUTING.md

```diff
@@ -13,6 +13,41 @@ test and deploy. Sometimes, however, HCO cannot guess what is the right thing to
 configurable is exposed in its Resource. Each configurable must be documented, so it is clear for a human operator when
 it should be used, and why the correct value cannot be guessed automatically.
 
+## Expectations from a PR
+
+The items below must be checked per PR by the author and reviewers. Authors are responsible for stating the status of PR and reviewers are responsible for checking/verifying them. 
+
+- ***PR Message:*** PR message must explain the purpose of the PR clearly. If it fixes a bug, describe the bug or mention the github issue/bugzilla id.
+If it adds a new feature, explain why we add it and how to use it.
+It is totally OK to copy/paste documentation into PR message.
+
+- ***Commit Messages:*** Commits in a PR are squashed and merged into the target branch as a single commit by kubevirt-bot. The message of that single commit consists of PR title and messages of all squashed commits. Therefore, commit messages must be clear, concise and comprehensive.
+
+
+- ***How to test:***: If automated tests are not applicable for the PR, explain how to test the functionality in the PR. 
+  
+- ***Unit Tests:*** Production code under `pkg` folder must have unit test. PRs should not decrease the test coverage. 
+
+- ***Functional Tests:*** New features must have functional tests under `tests/func-tests`. 
+  
+- ***User Documentation:*** If the PR adds/changes something which affect end users, user documents (e.g. /docs/api.md ) must be updated.
+  
+- ***Developer Documentation:*** If the PR adds/changes something which affect developers of this repo,developer documents (e.g. /docs/run-locally.md ) must be updated.
```

Comment:
very small nit: missing a space after the comma

Reply:
fixed. thx.